build/docker/docker-compose.yml

```diff
@@ -28,7 +29,7 @@ services:
     profiles: ["pg2pg"]
 
   kafka:
-    image: "bitnami/kafka:latest"
+    image: "bitnami/kafka:3.9.0"
```

Comment:
When I tried to run `pgstream run -c kafka2pg_tutorial.env --log-level trace`
I got the error - `Failed to join group pgstream-postgres-consumer-group: EOF`

The reason for the error seems to be - https://github.com/segmentio/kafka-go/issues/1360
segmentio/kafka-go does not seem to work with kafka 4.0 and using kafka 3.9.0 is working

Reply:
We've tried this on our end in different computers and we didn't encounter this problem 🤔. We can't really explain why this happens in your case, maybe there's some difference in the setup somewhere? 

In any case, we'd rather keep the version as up to date as possible if there's no reproducible issue with it, if you don't mind removing the change 🙏

Reply:
Thanks for checking, reverted the change.